src/Sql/Sql/Database/Cmdlet/RevalidateAzureSqlDatabaseTransparentDataEncryptionProtector.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.Azure.Commands.Sql.Database.Model;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.Database.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet for Revalidate Azure Sql Database encryption protector
+    /// </summary>
+    [Cmdlet("Invoke", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlDatabaseTransparentDataEncryptionProtectorRevalidation", SupportsShouldProcess = true), OutputType(typeof(AzureSqlDatabaseModel))]
+    public class RevalidateAzureSqlDatabaseTransparentDataEncryptionProtector : AzureSqlDatabaseCmdletBase<AzureSqlDatabaseModel>
```

Comment:
Change the class name and file name to match invoke-AzSqlDatabaseTransparentDataEncryptionProtectorRevalidation